modules/dnn/src/tengine4dnn/src/tengine_graph_convolution.cpp

```diff
@@ -34,8 +34,7 @@
 #ifdef HAVE_TENGINE
 
 #include "tengine_c_api.h"
```

Comment:
I see many Tengine lite calls in this file, and since this is C API now, I would consider adding some prefix to the functions and data structures names, e.g. teng_get_graph_node, teng_graph_t etc. Otherwise, without namespace and without any prefix the code is difficult to read and there is a big risk of name conflicts.

Reply:
done